Mod Menu Helper/assets/modmenu/lang/el_gr.json

```diff
@@ -35,6 +35,7 @@
   "modmenu.summaryTranslation.lithium": "Βελτίωση της φυσικής και κομματιών κόσμων\n§8✎ (μόνο εξιδικευμένες επιλογές)",
   "modmenu.summaryTranslation.isxander-main-menu-credits": "Έκδοση του πακέτου στην αρχική οθόνη\n§8✎ (μόνο εξιδικευμένες επιλογές)",
   "modmenu.summaryTranslation.memoryleakfix": "Διορθώνει διαρροές μνήμης RAM\n§8✎ (χωρίς επιλογές)",
+  "modmenu.summaryTranslation.midnightcontrols": "Υποστήρηξη χειριστηρίου και οθόνης αφής\n§9✎§8 Μέθοδος εισαγωγής, πλήκτρα, επιλογές χειριστηρίου, ...",
```

Comment:
First row is fine, second row last word is cut off

Reply:
Now?

Reply:
Oh, sorry, I meant that the entire word is gone.

Reply:
nvm yours fits
![image](https://user-images.githubusercontent.com/8611110/176999811-e61581ff-3429-4267-9da2-b2151aa2aaa3.png)

Reply:
Ok lemme remove some stuff and then u can merge it

Reply:
Ok done

Reply:
Well, some users may still use gui scale 2 or a different-sized display etc in which this could get displayed... but that's up to you :D

Reply:
The requirement say GUI Scale 3>, so I'll leave it this way :)